packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -196,6 +196,12 @@ define("$bootstrapModule", ["$entrypoint", "dart_sdk"], function(app, dart_sdk)
       return dart.getSourceMap(url);
     });
   }
+  // Prevent DDC's requireJS to interfere with modern bundling.
+  if (typeof define === 'function' && define.amd) {
+    // Preserve a copy just in case...
+    define._amd = define.amd;
```

Comment:
This bit is probably unnecessary. I'm not sure how anybody could be relying on this, when this is only exposed in `debug` mode (unless they're publishing the `debug` version of their web app, 😱)